src/main/java/athena/TaskList.java

```diff
@@ -173,4 +173,8 @@ public int getMaxIndex() {
     public void setMaxIndex(int maxIndex) {
         this.maxIndex = maxIndex;
     }
+
+    public Task at(int index) {
```

Comment:
Maybe a more descriptive method name would be helpful?

Reply:
Yea I think something like `getTask` would be nicer. Also there should be a JavaDoc comment.